NavfertyExcelAddIn/Commons/EnumerableExtensions.cs

```diff
@@ -50,7 +50,10 @@ public static void ForEachCell(this Range range, Action<Range> action)
             }
 
             // minimize number of COM calls to excel
-            var values = (object[,])rangeValue;
+            if (!(rangeValue is object[,] values))
+                return;
+
+            values = (object[,])rangeValue;
```

Comment:
```suggestion
```

Reply:
we do not need to reassign value - the "is" operator (`is object[,] values`) already did the whole work - declare, cast and assign casted value (or null)